src/Universe.cpp

```diff
@@ -1104,7 +1104,7 @@ void Lattice::setUniverses(int num_x, int num_y, Universe** universes) {
 
     _universes.push_back(std::vector< std::pair<int, Universe*> >());
 
-    for (int j = 0; j< _num_x; j++){
+    for (int j = 0; j < _num_x; j++){
```

Comment:
I feel like having `i` correspond to the x index and `j` correspond to the y index would be more conventional.